### PR TITLE
feat: adding playground to code snippet adds a playground link

### PR DESCRIPTION
**Pull request by ekeren**
if we add a playground to a code snippet in our docs, it will add a play button on the top of the link that will take you to the playground

https://github.com/winglang/docsite/assets/1727147/0c1b9d70-dcc9-47c2-b3cc-72427e579154

# Docs updates should not be submitted in this repository

Our doc site content is being automatically updated from [Wing](https://github.com/winglang/wing) repository docs folder.

Please submit any changes to the docs as a PR [here](https://github.com/winglang/wing/pulls)

### CODE REVIEW

File: src/theme/CodeBlock/Content/Element.js
Comment (ekeren) [inline src/theme/CodeBlock/Content/Element.js:1]: was created with `npm run swizzle @docusaurus/theme-classic CodeBlock/Content -- --eject`

File: src/theme/CodeBlock/Content/String.js
Comment (ekeren) [inline src/theme/CodeBlock/Content/String.js:1]: was created with `npm run swizzle @docusaurus/theme-classic CodeBlock/Content -- --eject`

File: src/theme/CodeBlock/Content/styles.module.css
Comment (ekeren) [inline src/theme/CodeBlock/Content/styles.module.css:1]: was created with `npm run swizzle @docusaurus/theme-classic CodeBlock/Content -- --eject`

**Review by ShaiBer — APPROVED** [review]
Really cool!!

File: src/theme/CodeBlock/Content/String.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <div className={styles.buttonGroup}>
+          {(wordWrap.isEnabled || wordWrap.isCodeScrollable) && (
+            <WordWrapButton
+              className={styles.codeButton}
+              onClick={() => wordWrap.toggle()}
+              isEnabled={wordWrap.isEnabled}
+            />
+          )}
+          
+          {(metastring ?? "").includes("playground") && <button className={clsx('clean-btn')} onClick={ () => window.open(`https://play.winglang.io?code=${Buffer.from(code).toString('base64')}`) }>Play</button>}
```
[the remark this answers is not in the document]
Comment (ekeren) [reply src/theme/CodeBlock/Content/String.js:94]: the rest was created with 
`npm run swizzle @docusaurus/theme-classic CodeBlock/Content -- --eject`

**Pull request merged**